content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2891,17 +2885,38 @@ In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Pri
 
 #### Examples
 
-In the following example, `Tag` is ignored because it does not have a valid unique identifier:
+In the following example, the `Post` model is invalid because it does not have a unique identifier. Use `@@ignore` to exclude it from the generated Prisma Client API:
 
-```prisma highlight=5;normal
+```prisma file=schema.prisma highlight=7;normal
 /// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
-model Tag {
-  id String? @db.VarChar
+model Post {
+  id       Int  @default(autoincrement()) // no unique identifier
+  author   User @relation(fields: [authorId], references: [id])
+  authorId Int
 
   @@ignore
 }
 ```
 
+In the following example, the `Post` model is invalid because it does not have a unique identifier, and the `posts` field is invalid because it refers to the invalid `Post` model. Use `@@ignore` on the `Post` model and `@ignore` on the `posts` field to exclude them from the generated Prisma Client API:
```

Comment:
```suggestion
In the following example, the `Post` model is invalid because it does not have a unique identifier, and the `posts` relation field on `User` is invalid because it refers to the invalid `Post` model. Use `@@ignore` on the `Post` model and `@ignore` on the `posts` relation field in `User` to exclude both the model and the relation field from the generated Prisma Client API:
```